src/Fable.Transforms/Rust/Fable2Rust.fs

```diff
@@ -421,11 +424,10 @@ module TypeInfo =
             | HasEmitAttribute _ -> None
             // do not make custom types Rc-wrapped by default. This prevents inconsistency between type and implementation emit
             | HasReferenceTypeAttribute ptrType ->
-                if not (isCopyableType com Set.empty typ) then
-                    Some ptrType
-                else None
-            | _ ->
-                if not (isCopyableType com Set.empty typ) then Some Lrc else None
+                Some ptrType
+            | ent ->
+                if ent.IsValueType then None
```

Comment:
@alexswan10k If `isCopyableType` already returns true for value types, is this needed?

Reply:
This is fine.